### PR TITLE
MWPW-130163: [Milo] Create version tag from milo doc

### PR DESCRIPTION
**Pull request by bandana147**
* Created a milo block version history which is used in creating a version history page
* That page will be added in cc repo config which will render the version history inside an iframe on the milo doc
* Added version creation flow when authors published a page through sidekick
* Version history button will be available under tools option in milo doc

Resolves: [MWPW-130163](https://jira.corp.adobe.com/browse/MWPW-130163)

**Test URLs:**
- [Version history page ](https://version-history--milo--bandana147.hlx.page/drafts/blaishram/document1?project=milo--adobecom&referrer=https%3A%2F%2Fadobe.sharepoint.com%2F%3Aw%3A%2Fr%2Fsites%2Fadobecom%2F_layouts%2F15%2FDoc.aspx%3Fsourcedoc%3D%257BF94A8B5A-E133-4F32-8C54-7AF0688C7CF0%257D%26file%3DVH.docx%26action%3Ddefault%26mobileredirect%3Dtrue%26cid%3D0d6ad2ab-0af0-4fbe-9156-35b32986cc22)

### CODE REVIEW

**Comment by bandana147** [comment]
Config file path will be changed after edit approval

File: libs/deps/msal-browser-2.34.0.js
Comment (bandana147) [inline libs/deps/msal-browser-2.34.0.js:1]: Added this file here to follow the folder structure that @auniverseaway has followed in the `locui` branch (current msal file under `tools/loc/libs`)
We can discuss if this needs to be placed elsewhere.

**Comment by sanrai** [comment]
@bandana147 the input text form as has no size limit. I would imagine there should be one added.

**Comment by bandana147** [comment]
> @bandana147 the input text form as has no size limit. I would imagine there should be one added.

The textarea which is the input form has a width of 96%.
Is this the input form that you are referring here?

**Comment by bandana147** [comment]
Closing this one and creating a new one from feature branch instead of main branch.

**Pull request closed without merging**